### PR TITLE
Update usaco-896.mdx

### PR DESCRIPTION
**Pull request by tmworz**
This pull request will fix some spacing issues (which I admittedly found cumbersome when coding)

### CODE REVIEW

**Comment by HarryWangATX** [comment]
ummmmmmmmmmmm

**Comment by HarryWangATX** [comment]
ig it helps formatting but....

**Comment by SansPapyrus683** [comment]
can... you at least refactor some of the code? (naming, comments, etc.)?

File: solutions/usaco-896.mdx [hunk 1]
```diff
@@ -88,6 +88,7 @@ public class mountains {
 			int e = x + y;
 			arr[i] = new mountain(s, e);
 		}
+
```
Comment (SansPapyrus683) [inline solutions/usaco-896.mdx:91]: and this line of whitespace isn't needed?

**Comment by tmworz** [comment]
> can... you at least refactor some of the code? (naming, comments, etc.)?

Done as asked...

**Comment by tmworz** [comment]
@SansPapyrus683 please review me sir

**Review by sathu-beep — CHANGES_REQUESTED** [review]
Your whitespaces actually make the code a lot harder to read than before. I would also ask you to remove your name from "author" as making extremely minor to almost non existent changes isn't enough to claim credit. Also I would mention that adding your name to the "author" section is the biggest change you made to this file. Capitalizing the first letter of an array also doesn't do anything...

File: solutions/usaco-896.mdx [hunk 2]
```diff
@@ -47,19 +47,25 @@ bool cmp(Mountain a, Mountain b) {
...
+	
 	int rightMost = -1, ans = 0;
+	
 	for(int i = 0; i < n; i++) {
 		if(mountains[i].r > rightMost) {
 			ans++;
 			rightMost = mountains[i].r;
 		}
 	}
+
```
Comment (SansPapyrus683) [inline solutions/usaco-896.mdx:68]: all of these lines of whitespace aren't necessary....

**Comment by HarryWangATX** [comment]
grammar has been fixed it #1702

**Pull request closed without merging**